Dockerfile

```diff
@@ -155,8 +156,10 @@ FROM compiler as builder
 ARG PG_VERSIONS
 
 # timescaledb-tune, as well as timescaledb-parallel-copy
+# TODO: Replace `focal` with `$(lsb_release -s -c)` once packages are available
+# for Ubuntu 21.10
 RUN wget -O - https://packagecloud.io/timescale/timescaledb/gpgkey | gpg --dearmor --output /usr/share/keyrings/timescaledb.keyring
-RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/timescaledb.keyring] https://packagecloud.io/timescale/timescaledb/ubuntu/ $(lsb_release -s -c) main" > /etc/apt/sources.list.d/timescaledb.list
+RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/timescaledb.keyring] https://packagecloud.io/timescale/timescaledb/ubuntu/ focal main" > /etc/apt/sources.list.d/timescaledb.list
```

Comment:
The former line seems more universal, why did we change it?

Reply:
Because there are no packages published for the Ubuntu codename `impish`, yet the `focal` packages are good enough, as we only install statically compiled binaries using this repository.

Once we switch to Ubuntu 22.04, we should be able to switch it back to the previous line, see the TODO added slightly above.

Reply:
Yeah, sorry, read the comment, but couldn't resolve the conversation, as the action failed.